common/src/main/java/com/scottlogic/deg/generator/utils/ListUtils.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 Scott Logic Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.scottlogic.deg.generator.utils;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ListUtils {
+
+    private ListUtils() {
+        throw new IllegalStateException("Do not instantiate utility class");
+    }
+
+    @SafeVarargs
+    public static <T> List<T> listOf(T... elements) {
```

Comment:
Arrays.asList does this exact thing right?

Reply:
It does! I'll change that from using the stream. The only thing this is adding is making the list immutable which we probably still want to do.